src/css/application.css

```diff
@@ -73,6 +73,90 @@ body {
   font-family: 'Roboto', sans-serif;
 }
 
+/** @define identity-confirmation */
+
+.identity-confirmation {
+  background: rgba(0, 0, 0, 0.48);
+  height: 100vh;
+  position: absolute;
+  width: 100vw;
+  z-index: 3;
+}
+
+.identity-confirmation__confirm-btn {
+  position: relative;
+
+  display: block;
+  margin: 30px auto;
+  padding: 0;
+
+  overflow: hidden;
+
+  border-width: 0;
+  outline: none;
+  border-radius: 2px;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, .6);
+
+  background-color: #2ecc71;
+  color: #ecf0f1;
+
+  transition: background-color .3s;
+}
+
+.identity-confirmation__confirm-btn:hover, .identity-confirmation__confirm-btn:focus {
+  background-color: #27ae60;
+}
+
+.identity-confirmation__confirm-btn {
+  cursor: pointer;
+  padding: 1em;
+}
+
+.identity-confirmation__confirm-btn > * {
+  position: relative;
+}
+
+.identity-confirmation__confirm-btn:before {
+  -moz-transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+  -o-transform: translate(-50%, -50%);
+  -webkit-transform: translate(-50%, -50%);
+  background-color: rgba(236, 240, 241, .3);
+  border-radius: 100%;
+  content: "";
+  display: block;
+  left: 50%;
+  padding: 0;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 0;
+}
+
+.identity-confirmation__confirm-btn:active:before {
+  padding-top: 120%;
+  transition: width .2s ease-out, padding-top .2s ease-out;
+  width: 120%;
+}
+
+.identity-confirmation__modal {
```

Comment:
So I think we actually want two separate BEM blocks here—a `modal` which deals with the basic modal layout, and then an `identity-confirmation` which styles the stuff in this particular modal. So it would look something like:

```html
<div class="modal>
  <div class="modal__contents identity-confirmation">
    <h1 class="identity-confirmation__title">Are you Eric Liu?</h1>
    <!-- etc. -->
  </div>
</div>
```